spec/controllers/admin_controller_spec.rb

```diff
@@ -94,7 +94,7 @@
             result_str << member1_info + ','
             result_str << '"",'  # no business categories
 
-            result_str << (m.company.nil? ? '' : '"' + m.company.name + '"')
+            result_str << (m.companies.any? ? '"' + m.companies.last.name + '"' : '')
```

Comment:
OK with me if it's OK with @thesuss .

It would be _best_ to make this specification explicit by having it be a separate spec/example.  I'll make a note in the PT feature that is outstanding that is about changing this code.  We can refactor this spec in that PR.